recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: pandas-{{ version }}.tar.gz
   url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
-  sha256: 78bb2b39c8a1ac508f640ac489deda34ffecf9989d07bf2ee800d9aa449f78d7
+  sha256: e0613cb3847d5e0eecf4735e067835c34f41c2f4b6e8d2f5ef01acd28db74c39
 
 build:
   number: 1
```

Comment:
Ahh shoot, ok.

Is it OK to let the CI finish first, change it, and then merge that commit? Don't want to put more strain on the CI servers than we have to. Or is there any harm in leaving it at 1?